src/lib/Microsoft.Health.Events/EventConsumers/Service/EventConsumerService.cs

```diff
@@ -51,5 +51,10 @@ public void NewPartitionInitialized(string partitionId)
         {
             // do nothing
         }
+
```

Comment:
Suggestion: Add logging messages to identify when the partition is stopped.

Reply:
the method is only needed because of the interface. Originally it was added to support multiple event consumers, but in practice we only have one consumer. I think the flow is: Processor (read events from event hub) -> Consumers (do something with them) and we do log that the partition is stopped in the Processor ([BaseEventProcessor.cs](https://github.com/microsoft/iomt-fhir/pull/241/files/1eb834f77a7ec97ca83fd469c3af76f1e996de7a#diff-21400b2a07723f26ba56809facbbcf88a8f0121ad9c14218d2891db69e8677a1)) so didn't feel like we needed to log it again here. We should probably delete the EventConsumerService class in the long run. It appears to still be used but am not sure it really serves a purpose since we only have one consumer and we should probably refactor it out.

Reply:
Thanks for the explanation Will.  Could you please encapsulate the details of what you are thinking in a User Story and send it to me?

Reply:
@dustinburson created work item 98220